layout/js/map.js

```diff
@@ -50,10 +47,28 @@ var animate_bars = function(data) {
     var bar = $("#bar-" + myYear);
     if (bar !== undefined) {
       bar.css(style);
+      if (myYear !== year) {
+        bar.css('opacity','0.25');
+      } else {
+        bar.css('opacity','1.0');
+      }
     }
   });
 };
 
+
+var setStateValue = function(state) {
+  $.get( "js/scaleFactors.json", function( data ) {
```

Comment:
We don't want an async call that gets called this often (it is probably hitting the cache which is why it works alright).  We could add in a promise from the onReady so we know we have the data (or just check that `transformData` is not undefined).